lib/matplotlib/tri/_tripcolor.py

```diff
@@ -163,5 +163,7 @@ def tripcolor(ax, *args, alpha=1.0, norm=None, cmap=None, vmin=None,
     corners = (minx, miny), (maxx, maxy)
     ax.update_datalim(corners)
     ax.autoscale_view()
-    ax.add_collection(collection)
+    # TODO: check whether the above explicit limit handling can be
+    # replaced by autolim=True
+    ax.add_collection(collection, autolim=False)
```

Comment:
Since there is an explicit `update_datalim(); autoscale_view()` above, `autolim=True` has no effect. Setting `autolim=False` for now to make that clear. We may whether the explicit handling can be removed in favor of `autolim=True`, but I'll leave that for later.